blocks/library/pullquote/index.js

```diff
@@ -48,9 +48,9 @@ export const name = 'core/pullquote';
 
 export const settings = {
 
-	title: __( 'Pullquote' ),
+	title: __( 'Pull quote' ),
```

Comment:
If it is two words, I'd be inclined to suggest updating the directory name from `pullquote` to `pull-quote`.

Reply:
That's true, although I don't know how much of a breaking change that'd be. I do see other blocks have a hyphen as well - probably best for consistency.

Reply:
I'm fairly certain we don't expose the path (or anything derived from the path) publicly, so it should be fine as long as all internal references are updated.